src/StatusCode/Vendor/Twitter.php

```diff
@@ -43,7 +38,14 @@
  * @link       http://shrikeh.github.com/teapot
  */
 
-interface Vendor extends Apache, Microsoft, Nginx, Twitter
```

Comment:
@shrikeh 

Seems that `Teapot\StatusCode\Vendor` as used in [`Teapot\StatusCode\All`](https://github.com/shrikeh/teapot/blob/master/src/StatusCode/All.php#L50) is now undefined, is that intended?